### DISCUSSION TITLE
add nextcloud-vfs as Cask?

### DISCUSSION THREAD

**Discussion by krombel** [Homebrew/discussions#5474] [Casks]
### Output of `brew config`

```shell
irrelevant
```

### Output of `brew doctor`

```shell
irrelevant
```

### Description of issue

The nextcloud client is currently released in 2 flavors:
- "standard" as it is currently in the repo as well and
- "vfs" which is a dedicated build which adds vfs support.

[Here](https://github.com/nextcloud/desktop/discussions/6656) is some more information on that. My question raises from the following statement:
> Please note that our current plans are to support both the standard sync client and the virtual files client concurrently. There are no plans at the moment to replace the standard client.

Does it make sense to add a cask like `nextcloud-vfs` which installs that flavoured build?

In the long term I expect the vfs functionality to land in the standard client but given the statement above it seems they have no time frame yet.

### Relevant casks

nextcloud

**Comment by p-linnane** [comment]
Seems fine to me. You can review https://docs.brew.sh/Adding-Software-to-Homebrew#casks for how to open a PR to add a new cask.

**Comment by krombel** [comment]
In the meantime nextcloud-vfs got added. So this is done
